src/plots/plot.ts

```diff
@@ -388,6 +388,7 @@ export class Plot extends Component {
    * Updates the extents associated with each attribute, then autodomains all scales the Plot uses.
    */
   protected _updateExtents() {
+    this._resetEntityStore();
```

Comment:
Yeah, we can probably remove it but previously we were clearing the cache here and i didn't want to miss it if it was needed. Since "clearing the cache" is just setting to `undefined` it's not gonna be slow